### ISSUE TITLE
[Moderate] Required field missing aria-required

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#615]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87777 |
| **Checkpoint** | 3.3 Input Assistance › Required Form Fields (3.3.2.c) |
| **Impact** | Moderate |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 New Job Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 5:15 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Best Practice |
| **Reported By** | kim.casey@ablr360.com |

## Description

The field is required but is missing aria-required="true"
Global: The required fields are indicated with an asterisk (*), but this is not announced to the screen reader. See Create Job User Experience video at 22:45.

## Recommendation

It is possible to have each field announce as required by using aria-required="true"

References:
aria-required attribute: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-required

## Screenshots

![03_required fields](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87777/03_required_fields.png)
